modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -633,14 +633,14 @@ public function enqueue_scripts() {
 		wp_enqueue_style(
 			'jetpack-admin-menu',
 			plugins_url( 'admin-menu.css', __FILE__ ),
-			array(),
-			'1'
+			defined( 'IS_WPCOM' ) && IS_WPCOM ? array( 'wpcom-admin-bar' ) : array( 'a8c-wpcom-masterbar' ),
+			20201215
```

Comment:
Would it be an option to switch to using Jetpack's version constant here and below?

```suggestion
			JETPACK__VERSION
```

Reply:
I was worried that `JETPACK__VERSION = wpcom` on simple sites. Generated URL
`https://cpapazoglousandbox.wordpress.com/wp-content/mu-plugins/masterbar/admin-menu/admin-menu.css?m=1607723919h&ver=wpcom` is `m=1607723919h` changing for each deployment? @obenland any clues?

Reply:
Yes, it's the result of `filemtime()`. See `wpcom_cdn_cache_bust_mtime()` on dotcom.

Reply:
Great! Resolved d3a61c4!